pkg/postgres/pgtest/postgres.go

```diff
@@ -127,6 +127,12 @@ func ForTCustomPool(t testing.TB, dbName string) *pgxpool.Pool {
 	return pool
 }
 
+// GetGormDB opens a Gorm DB to the Postgres DB
+func (tp *TestPostgres) GetGormDB(t testing.TB) *gorm.DB {
+	source := conn.GetConnectionStringWithDatabaseName(t, tp.database)
+	return OpenGormDB(t, source)
+}
+
```

Comment:
I don't see this being used anywhere, why do we need it?

Reply:
It is needed in the `access scope`/`permission set`/`role` migration test to create the `report_configurations` table, which is pre-populated then updated.
See the `SetupTest` function change.